features/mobile/reactions.feature

```diff
@@ -23,6 +23,17 @@ Feature: reactions mobile post
     And I click on selector "a.show-comments"
     Then I should see "1" within ".like-count"
 
+  Scenario: liking from the profile view
+    When I am on "alice@alice.alice"'s page
+    Then I should see "No reactions" within ".show-comments"
+    And I click on selector "span.show-comments"
```

Comment:
There is no event bind when there is no reactions. Even when the click is bind because there are comments, the `.like-action` is shown without having to open the panel. Can you please check if the test succeed without this line? If so, can you please remove it from this test and the one in the main stream, for both "No reaction" and "1 reaction" cases?